botfront/cypress/integration/02_training/training.spec.js

```diff
@@ -84,4 +84,36 @@ describe('Training', function() {
         cy.newChatSesh('fr');
         cy.testChatInput('salut', 'utter_hi');
     });
+
+    it('Should train and serve a model containing branches and links', function() {
+        cy.visit('/project/bf/settings');
```

Comment:
Identical to `importProject` in import-project.spec.js. Make that a Cypress command `importProject,` and call it from both specs.